app/presenters/pages_presenter.rb

```diff
@@ -128,8 +136,14 @@ def currently_sorting_by_key?(key)
   def top_phrases(page)
     phrases = Phrase.top_phrases_for_page(page[:page_id], start_date, end_date).take(3)
 
-    content_tag(:span, phrases.first, class: "phrase") +
-      content_tag(:span, phrases.second, class: "phrase") +
-      content_tag(:span, phrases.third, class: "phrase")
+    phrase_content_tag(0, phrases) +
+      phrase_content_tag(1, phrases) +
+      phrase_content_tag(2, phrases)
```

Comment:
I do believe that if `phrase_content_tag` fails to match anything on the `if`, it will return `nil` and this will throw an exception

Reply:
I made this refactor to fix that exact problem. For reasons that I actually don't fully understand, (and I've tested this) if passed an array of length one or two it's actually fine. Because saying "it works but I don't know why" isn't exactly best practice, I'll make it return a blank string anyway